rickroll.py

```diff
@@ -153,5 +174,55 @@ def original():
 
     return str(play_tune(tune))
 
+@app.route("/sms", methods=["POST"])
+def sms():
+    """Adds an incoming SMS to a queue, to reply to later."""
+    bucket.put_object(
+        Key='queue/{to}/{from_}'.format(to=request.form['To'],
+                                        from_=request.form['From']),
+        Body='',
```

Comment:
Does this lose the message they sent us? It *would* be visible in the Twilio logs, but in terms of maximum amusement for line operators, being able to keep the conversation in a more accessible form would be amazing. :)